The travis failure here is in `test.test_multiprocessing_forkserver.TestIgnoreEINTR`. Naively, that seems unlikely to be related to this change?

```
ERROR: test_ignore (test.test_multiprocessing_forkserver.TestIgnoreEINTR)
----------------------------------------------------------------------
Traceback (most recent call last):
  File "/home/travis/build/python/cpython/Lib/test/_test_multiprocessing.py", line 4359, in test_ignore
    os.kill(p.pid, signal.SIGUSR1)
ProcessLookupError: [Errno 3] No such process
```

--- a/Misc/NEWS.d/next/Library/2018-05-14-15-01-55.bpo-29235.47Fzwt.rst
+++ b/Misc/NEWS.d/next/Library/2018-05-14-15-01-55.bpo-29235.47Fzwt.rst
@@ -2,7 +2,7 @@ The :class:`cProfile.Profile` class can now be used as a context manager.
 You can profile a block of code by running::
 
   import cProfile
-  with cProfile.Profile as profiler:
+  with cProfile.Profile() as profiler:
       # ... code to be profiled ...
 
 Patch by Scott Sanderson.

